streamnative/README.md

```diff
@@ -0,0 +1,73 @@
+# StreamNative
+
+## Overview
+
+[StreamNative][1] provides an enterprise-grade messaging and event streaming platform built on Apache Pulsar. It offers scalable, real-time data streaming solutions with features like multi-tenancy, geo-replication, and seamless integration with cloud services.
+
+The StreamNative integration collects below types of [metrics][2]:
+
+1. Health
+2. Pulsar Resource
+3. Source Connector
+4. Sink Connector
+5. Kafka Connect
+
+## Setup
+
+### Configuration
+
+#### Get StreamNative Credentials
+
+1. Login to the [StreamNative Cloud Console Account][3].
+2. Get the `Organization ID` and `Instance Name`
+    - Click the profile icon and select the **Organizations** option.
+    - Select the **Organization** for which data needs to be collected.
+    - Obtain the **Instance Name** from the **Select an Instance** dropdown.
+3. Get the `Client ID` and `Client Secret`
+    - Click the profile icon and select the **Accounts & Accesses** tab.
+    - Select the Service Account for which **Admin** permission is **Enabled**.
+    - If the Service Account does not exist, click on **New -> Service Account** to create a new one, and ensure the **Super Admin** option is enabled.
+    - Click on `...` button on the right side of the selected Service Account.
+    - Click **Download OAuth2 Key** to get the **Client ID** and **Client Secret**.
+
+
+#### Add StreamNative Credentials
```

Comment:
Where is the user adding these?

Reply:
On Datadog Integration tile.

Reply:
Hi @jhgilbert , earlied we have had an discussion around this over slack with @dpoint01 ( [ref](https://crestdatasystems.slack.com/archives/C079S5829S9/p1729086353361489) ). Can you please have a look .

Reply:
I'm not able to see the ref thread, but I trust you if you prefer to leave the content as is. I'll resolve the conversation but feel free to reopen it if you want to discuss it further!